manifests/params.pp

```diff
@@ -218,7 +218,11 @@
     $server_root         = '/etc/apache2'
     $conf_dir            = $httpd_dir
     $confd_dir           = "${httpd_dir}/conf.d"
-    $conf_enabled        = "${httpd_dir}/conf-enabled"
+    if !($::operatingsystem == 'Ubuntu' and $::lsbdistrelease == '18.04') {
+      $conf_enabled      = "${httpd_dir}/conf-enabled"
+    } else {
+      $conf_enabled      = ''
```

Comment:
Could this line not go in the block for Ubuntu 18 at line 300, and then the original $conf_enabled value left untouched?